dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -1,12 +1,15 @@
 import { ArticleDesign } from '@guardian/libs';
 import { decideFormat } from '../web/lib/decideFormat';
 
-const enhanceCards = (collections: FEFrontCard[]): DCRFrontCard[] => {
-	const enhanced: DCRFrontCard[] = [];
-	collections.forEach((faciaCard) => {
-		if (faciaCard.format) {
+const enhanceCards = (collections: FEFrontCard[]): DCRFrontCard[] =>
+	collections
+		.filter(
+			(card: FEFrontCard): card is FEFrontCard & { format: CAPIFormat } =>
+				!!card.format,
+		)
```

Comment:
I’ve inlined the [type predicate](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates), because it’s not used elsewhere.